src/js/components/modals/ServicePlanProgressModalContents.js

```diff
@@ -0,0 +1,279 @@
+          <p className="service-plan-modal-footer-content short
+            flush-bottom">
+            We deployed the new configuration to
+            block {decisionPointBlock.getName()}. Please check your system
+            health and press Continue.
+          </p>
+        </div>
+      );
+    }
+
+    return (
+      <div className="service-plan-modal-footer">
+        {footerContent}
+        <div className="button-collection flush-bottom">
+          {this.getFooterActionItems(decisionPointBlock)}
+        </div>
+      </div>
+    );
+  }
+
+  getFooterActionItems(decisionPoint) {
+    let actionItems = [
+      <button className="button button-stroke" onClick={this.handleHideModal}
+        key="hide-button">
+        Hide
+      </button>
+    ];
+
+    if (!!decisionPoint) {
+      actionItems = actionItems.concat([
+        <button className="button button-danger" key="rollback-upgrade"
+          onClick={this.handleDecisionRollback}>
+          Rollback
+        </button>,
+        <button className="button button-success" key="continue-upgrade"
+          onClick={this.handleDecisionConfirm}>
+          Continue
+        </button>
+      ]);
+    } else {
+      actionItems.push(
+        <button className="button" onClick={this.handleUpgradePause}
+          key="pause-upgrade">
+          Pause Upgrade
+        </button>
+      );
+    }
+
+    return actionItems;
+  }
+
+  getPhaseProgress(servicePlan) {
+    return servicePlan.getPhases().getItems().map(function (phase) {
+      return {upgradeState: phase.status};
+    });
+  }
+
+  getProgressBarLabels(servicePlan, viewDetails) {
+    let planPhases = servicePlan.getPhases();
+    let phaseCount = planPhases.getItems().length;
+    let activePhaseIndex = planPhases.getActiveIndex();
+
+    return {
+      primaryTitle: `Phase ${activePhaseIndex + 1} of ${phaseCount}`,
+      secondaryTitle: (
+        <span>
+          {planPhases.getActive().getName()} {viewDetails}
+        </span>
+      )
+    };
+  }
+
+  getVersionNumber(service) {
+    return service.getMetadata().version;
+  }
+
+  render() {
+    let {service, servicePlan} = this.props;
+
+    if (!servicePlan) {
+      return null;
+    }
+
+    if (servicePlan.hasError()) {
+      return this.getErrorMessage();
+    }
+
+    if (servicePlan.isComplete()) {
+      return this.getCompleteMessage(service.getMetadata().name);
+    }
+
+    let detailsLabel = 'Show Details';
+    let modalContentClasses = classNames('service-plan-modal modal-content',
+      'allow-overflow', {
+        'is-paused': servicePlan.isPending()
+      });
+    let serviceHealth = service.getHealth();
+    let upgradeDetails;
+
+    if (this.state.detailsExpanded) {
+      detailsLabel = 'Hide Details';
+      upgradeDetails = (
+        <ServicePlanBlocks service={service} servicePlan={servicePlan} />
+      );
+    }
+
+    let hideShowDetails = (
+      <span>
+        (<a className="clickable" onClick={this.handleShowDetails}>
```

Comment:
Chris is correct, it's because the design team chose to render actual parentheses here. Thanks @daiyiwhale!